client/components/notes-formatted-block/test/blocks.js

```diff
@@ -167,9 +186,14 @@ describe( 'Comment block', () => {
 		};
 
 		const text = 'what a cool comment';
-		const comment = shallow( <Blocks.Comment content={ content } children={ text } /> );
+		render( <Blocks.Comment content={ content } children={ text } /> );
+
+		// @todo Not sure if I should use `getByText` or `queryByText` here.
```

Comment:
If we use `toBeVisible()` the assertion is no longer redundant.

Reply:
Ah, good point 👍🏻